lecture_02_data_structures_functions/hw/hw2.py

```diff
@@ -0,0 +1,22 @@
+"""
+Given an array of size n, find the max and min element.
+The max element is the element that appears more than ⌊ n/2 ⌋ times.
```

Comment:
May be rewrite it in python terms: 

```suggestion
The max element is the element that appears more than  n // 2 times.
```